src/loaders/trackedEntityLoader.js

```diff
@@ -91,7 +80,7 @@ const trackedEntityLoader = async (config) => {
 
     const fieldsWithRelationships = [...fields, 'relationships']
     // https://docs.dhis2.org/2.29/en/developer/html/webapi_tracked_entity_instance_query.html
-    let url = `/trackedEntityInstances?skipPaging=true&fields=${fieldsWithRelationships}&ou=${orgUnits}`
+    let url = `/tracker/trackedEntities?skipPaging=true&fields=${fieldsWithRelationships}&orgUnit=${orgUnits}`
```

Comment:
Should the comment from line 82 be removed?

Reply:
Addressed here: https://github.com/dhis2/maps-app/pull/3346/commits/2e17afcaab97cbce1cb836c554aecd31260f8d97